extra-js/nodejs/autobuild/defines

```diff
@@ -5,3 +5,6 @@ PKGDES="A JavaScript runtime built on the V8 JavaScript engine"
 
 PKGEPOCH=2
 NOLTO__LOONGSON3=1
+
+# LTO compilation with nodejs will fail
+NOLTO=1
```

Comment:
Consider just mark one `NOLTO`?

Reply:
Only this version off lto, will be repaired later, and Loongson3 off LTO is long-term, so retain NOLTO_LOONGSON3 = 1